libr/anal/esil_handler.c

```diff
@@ -57,6 +57,21 @@ R_API bool r_anal_esil_set_interrupt(RAnalEsil *esil, ut32 intr_num, RAnalEsilHa
 	return _set_interrupt (esil, intr, intr_num);
 }
 
+R_API RAnalEsilHandlerCB r_anal_esil_get_interrupt(RAnalEsil *esil, ut32 intr_num) {
+	r_return_val_if_fail (esil && esil->interrupts, NULL);
+	RAnalEsilHandler *handler = _get_interrupt (esil, intr_num);
+	return handler ? handler->cb : NULL;
+}
+
+R_API void r_anal_esil_del_interrupt(RAnalEsil *esil, ut32 intr_num) {
+	r_return_if_fail (esil && esil->interrupts);
+	if (!intr_num) {
```

Comment:
compare to == 0, using ! is the same, but we are actually comparing to a number, not a pointer or a boolean, using a number looks cleaner to me

Reply:
done